LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini

```diff
@@ -136,37 +184,67 @@ MIDSHORT_CONVENTIONAL_RANGE[13] = -8
 MIDSHORT_CONVENTIONAL_RANGE[14] = -12
 MIDSHORT_CONVENTIONAL_RANGE[15] = -16
 MIDSHORT_CONVENTIONAL_RANGE[16] = -20
+
+; MEDLONG WEAPONS: MARKSMANRIFLE
+MEDLONG_CONVENTIONAL_RANGE[0] = 0
+MEDLONG_CONVENTIONAL_RANGE[1] = -15
+MEDLONG_CONVENTIONAL_RANGE[2] = -13
+MEDLONG_CONVENTIONAL_RANGE[3] = -12
+MEDLONG_CONVENTIONAL_RANGE[4] = -10
+MEDLONG_CONVENTIONAL_RANGE[5] = -9
+MEDLONG_CONVENTIONAL_RANGE[6] = -6
+MEDLONG_CONVENTIONAL_RANGE[7] = -3
+MEDLONG_CONVENTIONAL_RANGE[8] = 0
+MEDLONG_CONVENTIONAL_RANGE[9] = 0
+MEDLONG_CONVENTIONAL_RANGE[10] = 0
+MEDLONG_CONVENTIONAL_RANGE[11] = 0
+MEDLONG_CONVENTIONAL_RANGE[12] = 0
+MEDLONG_CONVENTIONAL_RANGE[13] = 0
+MEDLONG_CONVENTIONAL_RANGE[14] = 0
+MEDLONG_CONVENTIONAL_RANGE[15] = 0
+MEDLONG_CONVENTIONAL_RANGE[16] = 0
 ; TODO
 
 [LWOTC_WeaponPack.X2Item_LaserWeapons]
 ; ***** ImagePaths *****
 AssaultRifle_Laser_ImagePath="UILibrary_LW_LaserPack.LaserRifle_Base"
+BattleRifle_Laser_ImagePath="UILibrary_LW_LaserPack.LaserRifle_Base"
 SMG_Laser_ImagePath="UILibrary_LW_LaserPack.LaserSMG_Base"
 Cannon_Laser_ImagePath="UILibrary_LW_LaserPack.LaserCannon_Base"
 Shotgun_Laser_ImagePath="UILibrary_LW_LaserPack.LaserShotgun_Base"
 SniperRifle_Laser_ImagePath="UILibrary_LW_LaserPack.LaserSniper_Base"
+MarksmanRifle_Laser_ImagePath="UILibrary_LW_LaserPack.LaserSniper_Base"
```

Comment:
Did realitymachina's pack not have these assets? If not I'll need to put in an art request. Let me know